src/components/Content.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { FilmTile } from "./FilmTile";
+
+export class Content extends React.PureComponent {
+    render() {
+        return(
+            <div className={"content"}>
+                {this.props.data.map(
+                    film => <FilmTile
```

Comment:
you forgot to add "key" property for iterate items.
https://stackoverflow.com/questions/28329382/understanding-unique-keys-for-array-children-in-react-js